App/templates/index.html

```diff
@@ -290,9 +286,9 @@ <h1 class="headroom text-center">metro extracts</h1>
       });
     var cities = countries.selectAll(".city").data(function(d){ return d.metros; });
     cities.enter().append("a").attr("class","city");
-    cities.text(function(d){ return d.name.split("_")[0].replace(/-/g," "); })
+    cities.text(function(d){ return d.name; })
       .attr("href",function(d){ 
```

Comment:
Okay looked at this a little more, and I’m not sure I understand why an href is being set at all. They’re being provided in the HTML loop on line 195; is this Javascript re-creating them from scratch? Can it instead hide/show as appropriate?